package.json

```diff
@@ -21,7 +21,7 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.6.0"
+    "ember-cli-babel": "^7.1.2"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
```

Comment:
We can do updates to ember-cli && ember-source next :)

Reply:
@snewcomer here you go https://github.com/poteto/ember-deep-set/pull/17